pallets/mining-rewards/Cargo.toml

```diff
@@ -13,40 +13,42 @@ publish = false
 targets = ["x86_64-unknown-linux-gnu", "aarch64-apple-darwin", "wasm32-unknown-unknown"]
 
 [dependencies]
-codec = { workspace=true, default-features = false, features = ["derive"] }
```

Comment:
I wonder why the formatting was still off here even after we did fmt...

Reply:
No

Reply:
Was kinda expecting fmt to also format toml files. But we could just taplo everything.